mycli/packages/expanded.py

```diff
@@ -8,6 +9,16 @@ def get_separator(num, header_len, data_len):
     sep = u"***************************[ %d. row ]***************************\n" % (num + 1)
     return sep
 
+def format_field(value):
+    # Returns the field as a text type, otherwise will hexify the string
+    try:
+        if isinstance(value, bytes):
+            return _text_type(value, "ascii")
+        else:
+            return _text_type(value)
+    except UnicodeDecodeError:
+        return _text_type('0x' + (codecs.getencoder('hex_codec')(value)[0]).decode('ascii'))
```

Comment:
I think we can use `binascii` to make this line shorter. This is purely aesthetics, so feel free to ignore if you don't agree. 

```
    return _text_type('0x' + binascii.hexlify(value).decode('ascii'))
```

You'll need to `import binascii` instead of `codecs`.